### ISSUE TITLE
No 'N' in the font: Liu Jian Mao Cao

### ISSUE THREAD

**Issue by sj-on** [smaranjitghose/doc2pen#322]
The character: capital N is missing from the font  Liu Jian Mao Cao. Attaching a crop for a reference.
![image](https://user-images.githubusercontent.com/46680993/103395933-92607e00-4b56-11eb-890a-577137f579a9.png)

**Comment by sj-on** [comment]
There are special characters like single quotes missing from some other fonts (especially the ones which are cursive) as well. The following happened to be a case with __Messed Font__.
![image](https://user-images.githubusercontent.com/46680993/103396081-1dda0f00-4b57-11eb-9c18-4a9738ef0db8.png)

**Comment by ashuvssut** [comment]
Do you know what is the reason behind these? Any guesses?

Google Fonts website shows that has these letters n characters exist. but still its not getting drawn on the canvas.

**Comment by sj-on** [comment]
> Do you know what is the reason behind these? Any guesses?
> 
> Google Fonts website shows that has these letters n characters exist. but still its not getting drawn on the canvas.

I am not entirely sure but  the UTF or ASCII code for 'N' might be messing up with the JS. I not sure. 😅

**Comment by smaranjitghose** [comment]
Thanks @sj-on for bringing this to our notice. Even I had not payed attention to this. Kudos to you for observing such an important yet hidden bug.

If you and @ashuvssut wish, you can collaborate and work to fix this.

I actually wanted a revamp of the fonts,
we don't need tons of them, just 7-15 shall suffice.

It would be great if you can manage to create and gather custom fonts using [this](https://www.calligraphr.com/en/) tool.

We shall officially acknowledge everyone who contributed a font. Even your friends who are not a part of this yet can volunteer. And every font can be added in a new PR.

Feel free to remove some redundant fonts that we are using as well

**Comment by ashuvssut** [comment]
@smaranjitghose I m gonna try debug it. First I have to learn some p5. It ud take some time to figure out. Can't exactly tell when I will be submitting the solution/PR.

**Comment by smaranjitghose** [comment]
> @smaranjitghose I m gonna try debug it. First I have to learn some p5. It ud take some time to figure out. Can't exactly tell when I will be submitting the solution/PR.

Take your time

**Comment by ashuvssut** [comment]
I still cant figure out why these characters are not getting drawn on the canvas.

@smaranjitghose I request you to assign someone new to work on this issue. I will now focus on contributing other issues.

**Comment by ashuvssut** [comment]
ref #329 to see what other fonts are there that are having this issue

**Comment by vismitap** [comment]
Hey! I had used the calligraphr tool to create a font for my college assignments and noticed the same issue. The reason for some letters not being displayed is this
![image](https://user-images.githubusercontent.com/47056243/103865466-44192500-50ea-11eb-9a94-c0c001c30a10.png)
I used a workaround by deleting some special characters from the end so that I at least get all the necessary alphabets and numbers. I suppose this issue cannot be solved by HTML, CSS and JS as this has nothing to do with the website but the tool itself.
